app/models/forum_post.rb

```diff
@@ -26,6 +26,12 @@ class ForumPost < ApplicationRecord
   after_update(:if => ->(rec) {rec.updater_id != rec.creator_id}) do |rec|
     ModAction.log(:forum_post_update, {forum_post_id: rec.id, forum_topic_id: rec.topic_id, user_id: rec.creator_id})
   end
+  after_update(:if => ->(rec) {rec.is_hidden? && rec.saved_change_to_is_hidden?}) do |rec|
```

Comment:
It would be cleaner to combine these two callbacks into a single callback and change the value passes as the first value to ModAction.log based on the current value of `is_hidden?`

Reply:
Yeah, you're right. Did it to this one and the other one as well.